src/main/java/com/sleekbyte/tailor/output/Printer.java

```diff
@@ -161,7 +188,17 @@ private void printParseErrorMessage() throws IOException {
 
     private long getNumMessagesWithSeverity(Severity severity) {
         return msgBuffer.values().stream()
-            .filter(msg -> !ignoredLineNumbers.contains(msg.getLineNumber()))
+            .filter(this::shouldIgnoreViolationMessage)
             .filter(msg -> msg.getSeverity().equals(severity)).count();
     }
+
+    private boolean shouldIgnoreViolationMessage(ViolationMessage msg) {
```

Comment:
Seems like the logic here is inverted. We should rename this method to `shouldDisplayViolationMessage`.